src/Modal.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace atk4\ui;
+
+class Modal extends View
```

Comment:
I can look into that. The idea of using Modal as a regular view was to conform with semantic modal behaviour, i.e. adding the content in DOM. 
```
$modal = $layout->add(['Modal', 'modal title go here']);
$msg = $modal->add([
    'Message',
    'This is a title of your message',
]);
$msg->text->addParagraph('You can add some more text here for your messages');

$layout->add('Button')->on('click', $modal->js()->modal('show'));
```